README.md

```diff
@@ -1 +1,30 @@
-# DSCI_522_Group411
\ No newline at end of file
+# Predicting Avocado Prices
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+*__Creators__: Katie Birchard, Ryan Homer, Andrea Lee*
+
+## Dataset
+
+We will be analyzing the [avocado prices dataset](https://www.kaggle.com/neuromusic/avocado-prices?fbclid=IwAR35kKP-Fz0yYZj-QqsZ6iNDSVnLBncxTOG3Cce3F5EupQTVHo85ecn7SBo) retrieved from Kaggle and compiled by the Hass Avocado Board using retail scan data. The dataset consists of approximately 18,000 records over 4 years (2015 - 2018). The dataset contains information about avocado prices, PLU numbers, types, region purchased, volume sold, bags sold, and date sold.
+
+## Research Question
+
+We will be answering the research question: **What is the strongest predictor of avocado prices?**
+
+Our goal is to find the feature that most strongly predicts the average price of avocados. A natural inferential sub-question would be to first determine if any of the features correlate with avocado prices. From our results, we can also compute a rank of features by importance.
+
+## Analysis
+
+To answer our research question, we will first need to determine our target and features. Our target will be `average price` and our features will be `type`, `region`, `date`, `total volume`, `4046`, `4225`, `4770`, `Total Bags`, `Small Bags`, `Large Bags`, and `XLarge Bags`.
```

Comment:
I don't think we will know the complete set of features until we build and test our model. For example, we may need to create a "month" feature.

Reply:
Also, in the EDA I have changed some of the features, such as making a column for PLU code and bag size. But I can change all that when I add the EDA section to the proposal.

Reply:
I have also created a month column and a year-month column.

Reply:
I added a another sentence in the proposal stating that our features may be subject to change.